mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.tsx

```diff
@@ -151,6 +155,29 @@ class ShowArtifactLoggedModelView extends Component<Props, State> {
     );
   }
 
+  validateModelForServingText(modelPath: any, servingInput?: string) {
+    if (servingInput === null) {
+      return `from mlflow.models import validate_serving_input\n
+model_uri = '${modelPath}'\n
+# The model logged does not contain input_example, you need to manually generate a serving payload
+from mlflow.models import convert_input_example_to_serving_input\n
+# Replace INPUT_EXAMPLE with your own input example to the model
```

Comment:
Should we say "replace" or just "define via assignment" the constant `INPUT_EXAMPLE`?

Reply:
updated to `define`!